lib/babe/verify.go

```diff
@@ -24,6 +24,8 @@ import (
 	babetypes "github.com/ChainSafe/gossamer/lib/babe/types"
 )
 
+var ErrProducerEquivocated = fmt.Errorf("block producer equivocated")
```

Comment:
should we start an `errors.go` file for the babe package?

Reply:
probably, I can move the errors from this file into an errors.go